client/src/components/qa/__tests__/qa.test.js

```diff
@@ -90,13 +108,61 @@ describe('properly renders an answer', () => {
     expect(date).toBeTruthy();
   });
   test('renders helpfulness count', () => {
-    render(<AnswerListEntry question={testAnswer} />);
-    const helpfulness = screen.getByText(/0/);
+    render(<AnswerListEntry answer={testAnswer} />);
+    const helpfulness = screen.getByText(/12/);
     expect(helpfulness).toBeTruthy();
   });
   test('renders report button', () => {
-    render(<AnswerListEntry question={testAnswer} />);
-    const report = screen.getByText(/Report/);
+    render(<AnswerListEntry answer={testAnswer} />);
+    const report = screen.getByRole('button', { name: 'Report' });
     expect(report).toBeTruthy();
   });
 });
+
+const testPhotos = [
+  {
+    id: 5347675,
+    url: 'http://res.cloudinary.com/fec-cars/image/upload/v1676073715/eaokljdt8oimivi0rk1b.jpg',
+  },
+  {
+    id: 5347679,
+    url: 'http://res.cloudinary.com/fec-cars/image/upload/v1676074347/hz09yvlenfgf9s66tlsh.jpg',
+  },
+];
+
+describe('image list properly renders', () => {
+  test('image list should render', () => {
+    render(<ImageList photos={testPhotos} />);
+    expect(screen.getByTestId('image-list')).toBeTruthy();
+  });
+  test('image entry should have a length of 2 if given test array', () => {
+    render(<ImageList photos={testPhotos} />);
+    expect(screen.getAllByTestId('image-entry')).toHaveLength(2);
+  });
+});
+
+const testPhoto = {
+  id: 5347675,
+  url: 'http://res.cloudinary.com/fec-cars/image/upload/v1676073715/eaokljdt8oimivi0rk1b.jpg',
+};
+
+describe('properly renders an image list entry', () => {
+  test('it renders a photo', () => {
+    render(<ImageListEntry photo={testPhoto} />);
+    const getPhoto = screen.getByAltText('5347675');
+    expect(getPhoto).toBeTruthy();
+  });
+});
+
+describe('buttons for modal windows properly render', () => {
+  test('Add question button renders', () => {
+    render(<AddQuestion product={40349} />);
+    const button = screen.getByRole('button', { name: 'Ask Your Question' });
+    expect(button).toBeTruthy();
+  });
+  test('Add answer button renders', () => {
+    render(<AddAnswer id={328752} />);
+    const button = screen.getByRole('button', { name: 'Add Your Answer' });
+    expect(button).toBeTruthy();
+  });
```

Comment:
Wish I could give you more suggestions here, but I'm still a bit rusty on testing. Gonna look into it more tonight! Looks good to me though